app.css

```diff
@@ -103,3 +103,350 @@ p {
+  background-image: url(./images/work.png);
+  background-repeat: no-repeat;
+  height: 372px;
+  justify-content: center;
+  margin: 5% 25% 5% 30%;
+}
+
+.work2title {
+  width: 295px;
+  height: 81px;
+  color: #fff;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-size: 32px;
+  line-height: 44px;
+  padding-top: 24px;
+  margin: 1% 10% 1% 10%;
+}
+
+.work2Description {
+  width: 301px;
+  height: 99px;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-top: 10px;
+  padding-left: 13px;
+  margin-top: 24px;
+}
+
+.work2Languages {
+  display: flex;
+  list-style: none;
+}
+
+.work2Button {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  flex-direction: row;
+  display: flex;
+  background: rgba(255, 255, 255, 0.24);
+  border: none;
+  color: #fff;
+  padding: 10px 12px;
+  align-items: center;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  margin: 32px 12px 12px 12px;
+}
+
+.work2Link {
+  background: #ff6b00;
+  color: #fff;
+  font-weight: 700;
+  width: 327px;
+  height: 48px;
+  border: none;
+  padding: 12px;
+  text-align: center;
+  cursor: pointer;
+}
+
+.work2Link:active {
+  background-color: #e05304;
+}
+
+.work2Link:hover {
+  border: #f55800;
+}
+
+.work2Link:disabled {
+  background-color: #f4f1f0;
+  border: none;
+  color: #5b5957;
+}
+
+.aboutMe {
+  background-color: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  padding: 120px  24px;
+  position: relative;
+}
+
+.aboutmeTitle {
+  width: 327px;
+  height: 79px;
+  color: #ff6b00;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+}
+
+.aboutMeDescription {
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-top: 12px;
+}
+
+.resumeContainer {
+  padding-top: 36px;
+}
+
+.resumebutton {
+  width: 156px;
+  height: 48px;
+  background-color: #ff6b00;
+  border: none;
+  color: #fff;
+  padding: 12px;
+  text-align: center;
+  display: inline-block;
+  font-weight: 400;
+  font-size: 15px;
+  cursor: pointer;
+  margin: -7% 6% 0% 6%;
+}
+
+.resumebutton:active {
+  background-color: #e05304;
+}
+
+.resumebutton:hover {
+  background-color: #fff5e1;
+  border: #f55800;
+  color: #f55800;
+}
+
+.resumebutton:disabled {
+  background-color: #f4f1f0;
+  border: none;
+}
+
+.background-img2 {
+  background: url('images/Group\ \(1\).png');
+  width: 160px;
+  height: 260px;
+  background-repeat: no-repeat;
+  position: absolute;
+  top: 370px;
+  right: 0;
+}
+
+.line2 {
+  padding-top: 200px;
+}
+
+.Language {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  margin-top: 10%;
+  justify-content: center;
+}
```

Comment:
- [ ] Please use` font-family: 'Inter';` on the tech categories as we are aiming for a project identical to the[ figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=ULC4CUKEW0pdVT14-0) so we must take into account every single detail including fonts, colors, margins, paddings, etc.

Figma Design:

<img width="147" alt="Screenshot 2023-04-01 at 10 07 43 AM" src="https://user-images.githubusercontent.com/44798044/229266860-ccce3ba7-41b2-4f8a-933d-6b5287a885ab.png">

Your Design

<img width="279" alt="Screenshot 2023-04-01 at 10 07 31 AM" src="https://user-images.githubusercontent.com/44798044/229266864-f5a98856-b363-4627-b7e0-5c08e8544bc6.png">